src/Core/src/Platform/Linux/WidgetColorExtensions.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using Gtk;
+using Microsoft.Maui.Graphics.Native.Gtk;
+
+namespace Microsoft.Maui
+{
+
+	public static class WidgetColorExtensions
+	{
+
+		public static void SetBackgroundColor(this Gtk.Widget widget, Graphics.Color color)
```

Comment:
.NET MAUI will have another object called Paint, that will give information about colors, gradients, etc. This extensions methods will be something like `SetBackground(this Gtk.Widget widget, Graphics.Paint paint)`, but for now is fine.

Reply:
so this is related to have a **Gtk** specific Microsoft.Maui.Graphics implementation?
is it https://github.com/dotnet/Microsoft.Maui.Graphics/blob/02fe4e30a2ca482351e21fbb74cf44235a89c9e6/src/Microsoft.Maui.Graphics/Paint.cs?

Reply:
Exactly